src/TrustedCreditor.sol

```diff
@@ -17,16 +17,6 @@ abstract contract TrustedCreditor {
 
     constructor() {}
 
-    /* //////////////////////////////////////////////////////////////
-                        LIQUIDATION LOGIC
-    ////////////////////////////////////////////////////////////// */
-
-    /**
-     * @notice Called by a vault when it is being liquidated (auctioned) to repay an amount of debt.
-     * @param debt The amount of debt that will be repaid after the auction.
-     */
-    function liquidateVault(uint256 debt) public virtual;
```

Comment:
why deleted ser?

Reply:
TrustedCreditor is a minimal implementation a trusted creditor should implement, so that they are compatible with Arcadia-Vaults. In the past a vault called liquidatevault on the TrustedCreditor -> it had to be part if the minimal implementation.

Now this is no longer the case -> can be removed.